src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -14,159 +14,133 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+          <optionalConform>
+            <orTerm>
+              <feature name="PIN"/>
+              <feature name="RID"/>
+              <feature name="FGP"/>
+              <feature name="FACE"/>
+            </orTerm>
+          </optionalConform>
+        </otherwiseConform>
+      </feature>
+      <feature bit="10" code="YDSCH" name="YearDayAccessSchedules" summary="Lock supports year day user access schedules">
+        <optionalConform/>
+      </feature>
+      <feature bit="11" code="HDSCH" name="HolidaySchedules" summary="Lock supports holiday schedules">
+        <optionalConform/>
+      </feature>
+      <feature bit="12" code="UBOLT" name="Unbolting" summary="Lock supports unbolting">
+        <optionalConform/>
+      </feature>
+      <feature bit="13" code="ALIRO" name="AliroProvisioning" summary="Lock supports Aliro credential provisioning as defined in [Aliro&amp;#93;">
+        <optionalConform/>
+      </feature>
+      <feature bit="14" code="ALBU" name="AliroBLEUWB" summary="Lock supports the Bluetooth LE + UWB Access Control Flow as defined in [Aliro&amp;#93;">
+        <optionalConform>
+          <feature name="ALIRO"/>
+        </optionalConform>
+      </feature>
+    </features>
+    <!-- The principle behind  mandatory/optional is the following:
          1. If attribute/command is marked as mandatory it should be mandatory here.
          2. If attribute/command is marked as optional it should be optional here.
          3. Everything that depends on a certain feature is optional because we have no way
             of setting up the dependencies here. Dependencies would be probably resolved in the
             cluster itself. Those attributes/commands are marked with a special comment. -->
-    
     <!-- Attributes -->
-    <attribute side="server" code="0" name="LockState" define="LOCK_STATE" type="DlLockState" min="0" max="3" isNullable="true" reportable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="1" name="LockType" define="LOCK_TYPE" type="DlLockType" min="0" max="11">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="2" name="ActuatorEnabled" define="ACTUATOR_ENABLED" type="boolean">
-      <mandatoryConform/>
-    </attribute>
+    <attribute side="server" code="0" name="LockState" define="LOCK_STATE" type="DlLockState" min="0" max="3" isNullable="true"/>
+    <attribute side="server" code="1" name="LockType" define="LOCK_TYPE" type="DlLockType" min="0" max="11"/>
+    <attribute side="server" code="2" name="ActuatorEnabled" define="ACTUATOR_ENABLED" type="boolean"/>
     <!-- Conformance feature DPS - for now optional -->
-    <attribute side="server" code="3" name="DoorState" define="DOOR_STATE" type="DoorStateEnum" min="0" max="5" isNullable="true" reportable="true" optional="true">
+    <attribute side="server" code="3" name="DoorState" define="DOOR_STATE" type="DoorStateEnum" min="0" max="5" isNullable="true" optional="true">
```

Comment:
Keep `reportable="true"` to align with spec.

Reply:
`reportable="true"` is completely ignored in this XML.